src/components/Tooltip.tsx

```diff
@@ -55,22 +56,19 @@ function Popper({ triggerRef, content, placement = "auto" }: PopperProps) {
   const [arrowRef, setArrowRef] = useState<HTMLDivElement | null>(null);
 
   const { styles, attributes } = usePopper(triggerRef.current, popperRef.current, {
+    strategy: "fixed",
```

Comment:
Will this cause the tooltip to stay in place when the user scrolls the page, or scrolls the trigger out of the viewport? I know the tooltip will disappear soon after the user is no longer hovering over the trigger, but just curious if the `fixed` strategy will prevent the Tooltip staying next to the trigger?

Reply:
We removed the fixed strategy and it still works